ONSdigital/eq-author-app
# Create authorPullRequest.yml

> BEFORE MAKING YOUR PR... There must be no linting errors and all tests must pass

### What is the context of this PR?

At a previous retrospective, we raised that a move to GitHub actions for running our continuous intergration tests may be necessary to speed up development time.

### How to review 

* Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
    * ensure it can be opened in Author;
    * then, ensure it can be viewed in Runner by pressing the **view survey** button
* Does this require a migration? We need one if existing JSON schema properties change
* See if the GitHub Action check does the same checks as we do for Travis

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process
